src/providers/userStoryCompletionProvider.ts

```diff
@@ -8,7 +8,8 @@ export class UserStoryCompletionProvider implements vsc.CompletionItemProvider {
 	constructor(private sessionStore: ISessionStore, private logger: Logger) {
 	}
 
-	async provideCompletionItems(document: vsc.TextDocument, position: vsc.Position, _token: vsc.CancellationToken, _context: vsc.CompletionContext) {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars

```

Comment:
See https://eslint.org/docs/rules/no-unused-vars#argsignorepattern

According to my quick research this should still work with Typescript:
`"@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" }]`

Then all these comments will be redundant. That's a common pattern to use _ prefix for arguments when not being used. What do you think?

Reply:
Indeed, my professional products are configured like that as well. Will update.